anyblok_furetui/furetui/crud.py

```diff
@@ -61,6 +69,15 @@ def read(cls, request):
         # check user is disconnected
         # check user has access rigth to see this resource
         model = request.params['context[model]']
+
+        if not cls.registry.FuretUI.check_acl(
+            request.authenticated_userid, model, "read"
+        ):
+            raise HTTPForbidden(
+                f"User '{request.authenticated_userid}' has to be granted "
+                f"'read' permission in order to read on "
+                f"model '{model}'."
+            )
         fields = cls.parse_fields(request.params['context[fields]'], model)
```

Comment:
maibe, it is better to put it in cls.parse_fields, because we could check the read on each model

Reply:
done !